README.md

```diff
@@ -1,2 +1,597 @@
-# cli
-Clean and elegant CLI development kit.
+# {Name}
+
+[![Build](https://github.com/candy-kingdom/cli/workflows/Build/badge.svg)](https://github.com/JoshuaLight/chalice-restful/actions)
+
+_Clean and elegant CLI development kit._
+
+## Overview
+
+{Name} is a microframework that allows building declarative and nice-looking CLI apps.
+
+Consider the following example as a demonstration of {Name} capabilities:
```

Comment:
Is _example_ and _demonstration_ a tautology?

Reply:
Generally speaking, I don't think so. But I clearly see that in this sentence these words are used inaccurately. Thanks for pointing out!

Changed to: _"Consider the following simple program as a demonstration of {Name} capabilities."_

Reply:
Fixed!